src/test/java/org/jabref/logic/integrity/BooktitleCheckerTest.java

```diff
@@ -25,4 +25,10 @@ void booktitleDoesNotAcceptsIfItEndsWithConferenceOn() {
     void booktitleIsBlank() {
         assertEquals(Optional.empty(), checker.checkValue(" "));
     }
+
+    @Test
+    void booktitleDoesNotAcceptFullURL() {
+        assertNotEquals(Optional.empty(), checker.checkValue("Proceedings of the https://example.com/conference"));
```

Comment:
Can you do a `@ParameterizedTest` and `@CsvSource` - so that the parameters are very easy to type?